tests/output/ConsoleInteractionTest.py

```diff
@@ -1082,3 +1082,25 @@ def test_source_lines(self):
                 {abspath('file'): ('def fun():\n', '    pass  \n')})
             self.assertEqual(stdout.getvalue(),
                              "('    pass  \\n',)\n")
+
+    def test_show_bears_specified_in_args(self):
+        from coalib import coala
+        from tests.TestUtilities import bear_test_module, execute_coala
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                'JavaTestBear')
+            self.assertEqual(retval, 0)
+            # 1 bear plus 1 line holding the closing colour escape sequence.
+            self.assertEqual(len(stdout.strip().splitlines()), 2)
+
+    def test_show_bears_specified_in_args_regex(self):
+        from coalib import coala
+        from tests.TestUtilities import bear_test_module, execute_coala
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                '*Java*', '*Space*')
```

Comment:
If possible (though not necessary now):
Can we make this more reliable? As when we add some bear with that name the test will fail.
Maybe it also helps to add a comment explaining what is likely failing so people not involved here immediately understand what has changed.

Reply:
I've modified tests added in this PR not to fail in such cases. I'm testing the name of the bear instead of number of bears now.